src/env-inl.h

```diff
@@ -146,7 +146,7 @@ inline bool Environment::AsyncHooks::pop_ids(double async_id) {
   if (uid_fields_[kCurrentAsyncId] != async_id) {
     fprintf(stderr,
             "Error: async hook stack has become corrupted ("
-            "actual: %'.f, expected: %'.f)\n",
+            "actual: %f, expected: %f)\n",
```

Comment:
> why is it a double?

So that it isn’t limited by 2^32 in its range, but still representable in JS. If you prefer, casting to `uint64_t` should be safe here.